app/scenes/select_server/select_server_container.js

```diff
@@ -5,16 +5,25 @@ import {bindActionCreators} from 'redux';
 
 import navigationSceneConnect from '../navigationSceneConnect';
 
-import {getPing} from 'service/actions/general';
-import {goToLogin} from 'app/actions/navigation';
+import {getPing, resetPing} from 'service/actions/general';
+import {RequestStatus} from 'service/constants';
 import * as SelectServerActions from 'app/actions/views/select_server';
 
 import SelectServer from './select_server';
 
 function mapStateToProps(state) {
+    const {config: configRequest, license: licenseRequest} = state.requests.general;
```

Comment:
Yeah, that makes a lot of sense. It'll save us a fair bit of typing in the long run, and keep things more consistent